src/components/payment/Final.jsx

```diff
@@ -0,0 +1,88 @@
+import { useState, useContext } from 'react';
+import { AuthContext } from '../../context/AuthContext';
+import { PaymentCont, SubmissionForm, UPI } from './Payment';
+import { Heading, Paragraph } from '../shared';
+import { Contactorg, Emaildata } from '../../data/paymentData';
+import { uploadReceipt } from './uploadReceipt';
+import { updateDocumentByUid } from '../../firebase/uploadProof';
+
+function Final() {
+	const { userInfo } = useContext(AuthContext);
+	var currentUser = userInfo[0];
+
+	const [file, setFile] = useState(null);
+
+	const uploadProof = async () => {
+		const proofURL = await uploadReceipt(file);
+		await updateDocumentByUid(currentUser.uid, proofURL);
+		await updateDocumentByUid(currentUser.uid, proofURL);
```

Comment:
Do we need this 2 times?

Reply:
removed . https://github.com/dscnitrourkela/project-udon/pull/33/commits/82dcf6ed3539e304f203451f64e4ef473a50a7cd